README.md

```diff
@@ -75,7 +75,7 @@ jobs:
       - name: Clone repo
         uses: actions/checkout@master
       - name: tfsec
-        uses: aquasecurity/tfsec-pr-commenter-action@v1.2.0
+        uses: aquasecurity/tfsec-pr-commenter-action@v1.3.1
```

Comment:
same here 
RE:  I will create a new release for this as v1.3.2, so we should point the docs to this new version as opposed to pointing to v1.3.1 which would be outdated.